actnow/settings.py

```diff
@@ -162,10 +162,18 @@
     "DEFAULT_AUTHENTICATION_CLASSES": [
         "rest_framework.authentication.TokenAuthentication",
         "rest_framework.authentication.SessionAuthentication",
+        "oauth2_provider.contrib.rest_framework.OAuth2Authentication",
     ],
 }
 OAUTH2_PROVIDER_APPLICATION_MODEL = "accounts.ActNowApplication"
-
+OAUTH2_PROVIDER = {
+    # this is the list of available scopes
+    "SCOPES": {
+        "read": "Read scope",
+        "write": "Write scope",
+        "groups": "Access to your groups",
```

Comment:
What kind of access to groups i.e. read, write, etc.?

Reply:
>What kind of access to groups
Both read and write. But these aren't the final SCOPES. I think we need to agree on them first.